src/accelerate/accelerator.py

```diff
@@ -862,6 +862,52 @@ def no_sync(self, model):
         with context():
             yield
 
+    @staticmethod
+    @contextmanager
+    def ddp_trigger_sync_in_bwd(model_ddp):
```

Comment:
```suggestion
    def trigger_sync_in_backward(model):
```
I don't think we need to put the DDP in the name.